packages/plugin-ext/src/plugin/status-bar/status-bar-item.ts

```diff
@@ -41,6 +41,9 @@ export class StatusBarItemImpl implements theia.StatusBarItem {
         priority: number = 0) {
         this._proxy = _proxy;
         this._alignment = alignment;
+        if (alignment === StatusBarAlignment.Right && priority <= 0) {
```

Comment:
It could break order between plugin items. Cannot we make sure that `Toogle Bottom Panel` is always most right instead?

Reply:
I reverted the previous commit, and instead i moved the `core.toggle.bottom.panel` to the last index during render. This way, it wont mess up any other plugins and the `Toogle Bottom Panel` will always be the last item(most right). 

Another approach would be via css, setting the `Toogle Bottom Panel` position to absolute and setting the correct top, right etc.. But i think the solution above is more practical and less prone to error